tests/test_postgresql.py

```diff
@@ -50,6 +50,11 @@ def test_rand_postgres_port(postgresql_rand):
     assert postgresql_rand.status == psycopg2.extensions.STATUS_READY
 
 
+def test_rand_postgres_port(postgresql_password):
```

Comment:
I think I'd love specific executor test, but I already have one on other Pull request. This is great as well!

Reply:
Just the name overshadows the previous one

Reply:
Oh, yes, sorry. Didn't see that. Fixed.